client/landing/jetpack-cloud/sections/settings/main.jsx

```diff
@@ -10,66 +10,87 @@ import { localize } from 'i18n-calypso';
+
+		const iconPath = isConnected ? connectedIcon : disconnectedIcon;
+		const myClassName = isConnected ? 'settings__connected' : 'settings__disconnected';
+		const title = isConnected
+			? translate( 'Server status: Connected' )
+			: translate( 'Server status: Not connected' );
+		const content = isConnected
+			? translate( 'One-click restores are enabled.' )
+			: translate(
+					'Enter your server credentials to enable one-click restores for Backups. {{a}}Find your server credentials{{/a}}',
+					{
+						components: {
+							a: (
+								<ExternalLink
+									className="settings__link-external"
+									icon
+									href="https://jetpack.com/support/adding-credentials-to-jetpack/"
+								/>
+							),
+						},
+					}
+			  );
+
+		return (
+			<Card compact={ true } className={ myClassName }>
+				<img className="settings__icon" src={ iconPath } alt="" />
+				<div className="settings__details">
+					<div className="settings__details-head"> { title } </div>
+					<div>{ content }</div>
+				</div>
+			</Card>
+		);
+	}
+
 	render() {
 		const { rewind, siteId, translate } = this.props;
 
-		const isConnected = 'active' === rewind.state;
+		const isInitialized = ! ( 'uninitialized' === rewind.state );
 
 		return (
 			<Main className="settings">
 				<DocumentHead title={ translate( 'Settings' ) } />
 				<SidebarNavigation />
 				<QueryRewindState siteId={ siteId } />
 				<PageViewTracker path="/settings/:site" title="Settings" />
-				<div className="settings__page-title">{ translate( 'Server connection details' ) }</div>
-				{ isConnected && (
-					<Card compact={ true } className="settings__connected">
-						<Gridicon icon="checkmark-circle" />
-						<div className="settings__details">
-							<div className="settings__details-head">
-								{ translate( 'Server status: Connected' ) }
-							</div>
-							<div>{ translate( 'One-click restores are enabled.' ) }</div>
-						</div>
-					</Card>
-				) }
-				{ ! isConnected && (
-					<Card compact={ true } className="settings__disconnected">
-						<Gridicon icon="cross-circle" />
-						<div className="settings__details">
-							<div className="settings__details-head">
-								{ translate( 'Server status: Not connected' ) }
-							</div>
-							<div>
-								{ translate(
-									'Enter your server credentials to enable one-click restores for Backups. Find your server credentials.'
-								) }
-							</div>
-						</div>
-					</Card>
-				) }
-				<RewindCredentialsForm
-					{ ...{
-						allowCancel: false,
-						role: 'main',
-						siteId,
-						showNotices: false,
+				<div className="settings__title">
+					<h2>{ translate( 'Server connection details' ) }</h2>
+				</div>
+
+				{ isInitialized && this.renderServerConnectionStatus() }
+
+				{ ! isInitialized && <div className="settings__is-uninitialized" /> }
+
+				<ServerCredentialsForm
+					role="main"
+					siteId={ siteId }
+					labels={ {
+						save: translate( 'Save credentials' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Save credentials and fix' )` *ES Score: 7*